server/src/main/java/com/paypal/selion/grid/FileExtractor.java

```diff
@@ -70,7 +71,8 @@ static String getFileNameFromPath(String name) {
         }
         default: {
             Collections.addAll(executableNames, ProcessNames.PHANTOMJS.getWindowsImageName(),
-                    ProcessNames.CHROMEDRIVER.getWindowsImageName(), ProcessNames.IEDRIVER.getWindowsImageName());
+                    ProcessNames.CHROMEDRIVER.getWindowsImageName(), ProcessNames.IEDRIVER.getWindowsImageName(),
+                    ProcessNames.EDGEDRIVER.getWindowsImageName());
```

Comment:
I know this is not your change but will you please flip this `switch` statement. Let's default to `nix` instead of `windows`